htdocs/class/textsanitizer/syntaxhighlight/syntaxhighlight.php

```diff
@@ -48,7 +48,7 @@ public function load($ts, $source, $language)
      *
      * @return mixed|string
      */
-    public function php($text)
+    public static function php($text)
```

Comment:
The caller in this case should be modified instead. It should use $this->php(...) instead of the static call.